cmd/sops/main.go

```diff
@@ -192,6 +195,10 @@ func main() {
 			Name:  "encryption-context",
 			Usage: "comma separated list of KMS encryption context key:value pairs",
 		},
+		cli.StringFlag{
+			Name:  "set",
+			Usage: `set a specific key or branch in the input JSON or YAML document. value must be a json encoded string. (edit mode only). eg. --set '["somekey"][0] {"somevalue":true}'`,
```

Comment:
Oh, I just copied that from the python version https://github.com/mozilla/sops/blob/python-sops/sops/__init__.py#L148-L153